<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinecraftForm</class>
 <widget class="QMainWindow" name="MinecraftForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinecraftForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gridGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="formWrapper">
       <property name="title">
        <string>Add box </string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>286</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="formLayout">
         <item row="0" column="0" colspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>World position:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSpinBox" name="xInput">
           <property name="minimum">
            <number>-214748364</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="xLabel">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="yInput">
           <property name="minimum">
            <number>-214748364</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="zInput">
           <property name="minimum">
            <number>-214748364</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="zLabel">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Z</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="yLabel">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Block type:</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="materialInput">
           <item>
            <property name="text">
             <string>Dirt</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Glass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grass</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QPushButton" name="submit">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color=rgb(85, 170, 98)</string>
           </property>
           <property name="text">
            <string>Add new block</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="listWrapper">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>321</width>
          <height>213</height>
         </rect>
        </property>
        <property name="title">
         <string>Lists of blocks</string>
        </property>
        <widget class="QTableWidget" name="list">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>291</width>
           <height>181</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Block type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="font">
           <font>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="0">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>180</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Block size</string>
        </property>
        <widget class="QDoubleSpinBox" name="blockSizeInput">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMinecraft_world_editor">
    <property name="title">
     <string>Minecraft world editor</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSaveAsJson"/>
    <addaction name="actionSaveAsObj"/>
   </widget>
   <addaction name="menuMinecraft_world_editor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionHideList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionHideForm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionSaveAsJson">
   <property name="text">
    <string>Save as json</string>
   </property>
  </action>
  <action name="actionSaveAsObj">
   <property name="text">
    <string>Save as obj</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
